src/merge.rs

```diff
@@ -123,5 +213,23 @@ fn merge_with_zero<H: Hasher + Default>(
                 zero_count: zero_count.wrapping_add(1),
             }
         }
+        #[cfg(feature = "trie")]
+        MergeValue::ShortCut {
+            key,
+            value,
+            height: _,
```

Comment:
```suggestion
            ..
```
Otherwise may confused with `height` value in the function parameter.